qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +367,55 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def particle_number(n_orbitals, mapping="jordan_wigner"):
```

Comment:
No need to count the frozen electrons populating the core orbitals, this makes ``particle_number`` simpler without any lost of generality.

Reply:
Wow, this is huge improvement since I last looked!